remote-config.jsonc

```
@@ -16,50 +16,53 @@
 			"interval": 20,
 			"messages": [
 				{
-					"message": "Please encourage your organization to support DDEV financially, see https://github.com/sponsors/ddev"
+					"message": "Please consider supporting DDEV: https://ddev.com/support-ddev/"
```

```suggestion
					"message": "Please consider supporting DDEV financially: https://ddev.com/support-ddev/"
```

I love ddev.com/support-ddev but it's fairly hard to get to the financial part there. In fact I just read the page and skipped over it. I see the nice list of sponsors but don't get the call to financial support. I guess that's something we can work on over there.

---

The page was meant to encourage and channel any interest in supporting DDEV, not limited to financial sponsorship since not everyone is able to contribute in that way. My understanding is that the project could always use stars, community support, and different kinds of involvement. Hence this separate, not-financially-specific message vs. the one aimed at financial sponsorship.

---

It's absolutely true of course. But we need people to think in terms of concrete actions, and I don't feel like we get people to immediate, concrete actions over there. We get them aware (if they scroll all the way through) that there are many ways to get involved, but we don't get them started.

---

If this message + link aren’t working for you, I’d get rid of it. No need to duplicate messages specifically for sponsorship.

---

> But we need people to think in terms of concrete actions, and I don't feel like we get people to immediate, concrete actions over there.

Each call to action is followed by a large button labeled with a specific action. What would be more specific or immediate?

---

Eh, nevermind I’ll just remove it.